Contributor/En/Guide.xyl

```diff
@@ -392,6 +392,40 @@ $ git commit</code></pre>
   <p>Do not edit the <code>CHANGELOG.md</code> file. It is automatically
   generated.</p>
 
+  <h3 id="Commit_message_structure" for="main-toc">Commit message structure</h3>
+
+  <p>Has we already seen in the previous section, a commit title must be less
+  than 50 characters long. This title must be precise and reading it must allow
+  to identify the corresponding modifications.</p>
+
+  <p>To facilitate the writing of this title, we defined a structure:
+  <code>type(scope) title</code><p>
+
+  <p>This structure allow to describe very easily which kind of commit is it and
+  on which part. We have indeed isolated several recurring types in a
+  project:</p>
+
+  <ul>
+    <li><code>feat</code> for a new feature,</li>
+    <li><code>fix</code> for a bug fix,</li>
+    <li><code>test</code> for a test modification,</li>
+    <li><code>doc</code> for a documentation modification,</li>
+    <li><code>chore</code> to replace the “Quality” type.</li>
+  </ul>
+
+  <p>For example, we can obtain:</p>
+
+  <ul>
+    <li><code>feat(permission) Search backward bla bla.,</code></li>
+    <li><code>fix(user) `getName` is incorrectly computing its value.,</code></li>
+    <li><code>…</code></li>
+  </ul>
+
+  <p>The goal is to generate a <code>CHANGELOG</code> file as suggested by
+  <a href="http://keepachangelog.com/fr/1.0.0/">http://keepachangelog.com/fr/1.0.0/</a>,
```

Comment:
Not sure we want the french link here, prefer http://keepachangelog.com/en/1.0.0/. Also the link text should be “Keep a Changelog”.